app/components/FAQ.tsx

```diff
@@ -15,15 +15,11 @@ interface FAQProps {
 export function FAQ({ faqs, description = "" }: FAQProps) {
 	const renderQuestion = (faq: FAQ) => {
 		return (
-			<div key={faq.id} className={`${orangeInfoCardClasses} collapse-plus collapse border`}>
+			<div key={faq.id} className={`${orangeInfoCardClasses} collapse-plus collapse`}>
 				<input type="checkbox" />
-				<div className="collapse-title text-primary text-lg font-medium">{faq.question}</div>
+				<div className="collapse-title text-base-content text-lg font-medium">{faq.question}</div>
```

Comment:
Instead of specify color here (for every element), we can move it to the parent: `<div className="space-y-4">{faqs.map(renderQuestion)}</div>` (line 36)